.env.template

```diff
@@ -0,0 +1,35 @@
+# .env file
+ANTHROPIC_API_KEY=your_anthropic_api_key_here
```

Comment:
Remove the values, just keep the key so that when loading it with dotenv it will be None.

Reply:
Done